Theme.scss

```diff
@@ -7517,6 +7532,18 @@ a.btn,
     background-color: #bd4a29 !important;
 }
 
+.user-status-message-wrapper div {
+    color: black !important;
```

Comment:
```suggestion
    color: white !important;
```

Reply:
Oh i see why this was done.... profile message has white background. so white wasnt visible